Realm/RLMUtil.mm

```diff
@@ -429,11 +433,32 @@ id RLMMixedToObjc(realm::Mixed const& mixed) {
         case realm::type_OldTable:
         case realm::type_OldDateTime:
             REALM_UNREACHABLE();
+        case realm::type_UUID:
+            return [[NSUUID alloc] initWithRealmUUID:mixed.get<realm::UUID>()];
         default:
             @throw RLMException(@"Invalid data type for RLMPropertyTypeAny property.");
     }
 }
 
+realm::UUID RLMObjcToUUID(__unsafe_unretained id const value) {
+    try {
+        if (!value || value == NSNull.null) {
+            return realm::UUID(realm::null());
```

Comment:
realm::UUID(realm::null()) appears to just throw an exception

Reply:
Thanks. this is shouldn't be there